src/gui/editors/TrackContainerView.cpp

```diff
@@ -500,8 +502,17 @@ void TrackContainerView::scrollArea::wheelEvent( QWheelEvent * _we )
 	}
 }
 
+int TrackContainerView::totalHeightOfTracks()
+{
+	unsigned int heightSum = 0;
```

Comment:
The return type is `int`, but the sum is in `unsigned int`, which looks inconsistent.